### PR TITLE
added: modal component

### PR DESCRIPTION
**Pull request by sharmayatendra**
# Modal

## Open for review

image

### CODE REVIEW

**Review by vikrantkode — COMMENTED** [review]
Looks good to me. Can merge

File: components/Modal/modal.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                    </div>
+                </div>
+                <iframe class="code-snippet"
+  src="https://carbon.now.sh/embed?bg=rgba%287%2C94%2C84%2C1%29&t=night-owl&wt=none&l=htmlmixed&width=438&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=171%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522modal-container%2522%253E%250A%2520%2520%253Cdiv%2520class%253D%2522modal-header%2522%253E%250A%2520%2520%2520%2520%2520%2520%253Ch2%2520class%253D%2522modal-heading%2522%253EModal%2520Header%253C%252Fh2%253E%250A%2520%2520%2520%2520%2520%2520%253Cp%2520class%253D%2522modal-text%2522%253EI%2520am%2520modal%2520text.%253C%252Fp%253E%250A%2520%2520%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522alert-btn-container%2522%253E%250A%2520%2520%2520%2520%2520%2520%253Cbutton%2520class%253D%2522alert-btn%2522%253EACTION%25201%253C%252Fbutton%253E%250A%2520%2520%2520%2520%2520%2520%253Cbutton%2520class%253D%2522alert-btn%2522%253EACTION%25202%253C%252Fbutton%253E%250A%2520%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
```
Comment (bishnoimukesh) [inline components/Modal/modal.html]: take care of formatting

**Review by bishnoimukesh — APPROVED** [review]
suggestion given rest look good to go.

File: components/Modal/modal.html (hunk 1 above)
Comment (sharmayatendra) [reply components/Modal/modal.html]: done

**Pull request merged**